src/reducers/systemIntakesReducer.test.ts

```diff
@@ -0,0 +1,85 @@
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchSystemIntakes } from 'types/routines';
+import systemIntakesReducer from './systemIntakesReducer';
+
+describe('The system intakes reducer', () => {
+  it('returns the initial state', () => {
+    expect(systemIntakesReducer(undefined, {})).toEqual({
+      systemIntakes: [],
+      error: null,
+      isLoading: null,
+      loadedTimestamp: null
+    });
+  });
+  it('handles fetchSystemIntakes.REQUEST', () => {
+    const mockRequestAction = {
+      type: fetchSystemIntakes.REQUEST,
+      payload: undefined
+    };
+
+    expect(systemIntakesReducer(undefined, mockRequestAction)).toEqual({
+      systemIntakes: [],
+      error: null,
+      isLoading: true,
+      loadedTimestamp: null
+    });
+  });
+  it('handles fetchSystemIntakes.SUCCESS', () => {
+    const mockApiSystemIntake = {
```

Comment:
This is actually the data that's returned from the system intake. It's structured a bit differently.

Reply:
Instead of replacing with `initialSystemIntakeForm` we might consider seeding this with some filled in data, or make this a var outside of the tests and edit the data inside each test to test for various saved responses to match.